lib/src/base-styles/_nx-card.scss

```diff
@@ -6,176 +6,74 @@
-  }
-
-  .nx-card__call-out--text-only {
-    + .nx-card__content {
-      margin-top: $nx-spacing-xs;
-    }
-  }
-
-  .nx-card__footer {
-    border-top: $nx-border-default;
-    margin-top: $nx-spacing-l;
-    padding-top: $nx-spacing-l;
-    width: 100%;
-
-    > .nx-btn-bar {
-      justify-content: center;
-    }
-  }
+  flex-direction: column;
+  flex-grow: 1;
+  gap: $nx-spacing-md;
 }
 
-.nx-card-container--column {
-  @include container-vertical;
+.nx-card__footer {
+  border-top: $nx-border-default;
+  margin-top: $nx-spacing-l;
+  padding-top: $nx-spacing-l;
 
-  .nx-card {
-    display: -ms-grid;
-    display: grid;
-    -ms-grid-columns: auto 1fr auto;
-    grid-template-columns: auto 1fr auto;
-    -ms-grid-rows: auto auto;
-    grid-template-areas:
-      "card-callout card-title card-footer"
-      "card-callout card-text card-footer";
+  > .nx-btn-bar {
+    justify-content: center;
   }
+}
 
-  .nx-card__header {
-    -ms-grid-row: 1;
-    -ms-grid-column: 2;
-    grid-area: card-title;
-
-    @include container-vertical;
-    margin: 0 0 $nx-spacing-md 0;
-    padding: 0;
-  }
+// Common card Helpers
+.nx-card--equal {
+  flex: 1;
+  max-width: unset;
+}
 
-  .nx-card__content {
-    -ms-grid-row: 2;
-    -ms-grid-column: 2;
-    grid-area: card-text;
-    margin: 0;
-    padding: 0;
-  }
+.nx-card__call-out {
+  @include bold();
 
-  .nx-card__call-out {
-    -ms-grid-row-align: center;
-    align-self: center;
-    -ms-grid-row: 1;
-    -ms-grid-row-span: 2;
-    -ms-grid-column: 1;
-    grid-area: card-callout;
-    -ms-grid-column-align: center;
-    justify-self: center;
+  font-size: 32px;
 
-    align-items: center;
-    border-right: $nx-border-default;
-    display: flex;
-    height: 100%;
-    margin: 0 $nx-spacing-xxl 0 0;
-    padding: 0 $nx-spacing-xxl 0 0;
+  .nx-card__call-out-icon {
+    font-size: 48px;
   }
 
-  .nx-card__footer {
-    box-sizing: content-box;
-    -ms-grid-row-align: center;
-    align-self: center;
-    -ms-grid-row: 1;
-    -ms-grid-row-span: 2;
-    -ms-grid-column: 3;
-    grid-area: card-footer;
-    -ms-grid-column-align: start;
-    justify-self: start;
-
-    @include container-horizontal;
-    align-items: center;
-    border: none;
-    border-left: $nx-border-default;
-    display: flex;
-    height: 100%;
-    margin: 0 0 0 $nx-spacing-xxl;
-    padding: 0 0 0 $nx-spacing-xxl;
-    // necessary to make the link wrap
-    max-width: 150px;
+  .nx-card__call-out-icon--xl {
+    font-size: 64px;
   }
 }
 
-// Styles for the alternate row layout
-.nx-card-container--row {
-  .nx-card__content-container {
-    align-items: center;
-    display: flex;
-
-    .nx-card__call-out {
-      @include container-horizontal;
-      flex-grow: 1;
-      text-align: right;
-
-      + .nx-card__content {
-        margin-left: $nx-spacing-l;
-      }
-    }
-
-    .nx-card__content {
-      flex-grow: 1;
-      margin-top: 0;
-      text-align: left;
-    }
-  }
+.nx-card__content--2-column {
```

Comment:
I'm mildly concerned about the naming of this class in light of the fact that we explicitly support using other things as the `.nx-card__content` children besides `.nx-card__callout` and `.nx-card__text`.  It's only really a 2-column layout when the `.nx-card__content` happens to have two children.

Reply:
> I'm mildly concerned about the naming of this class in light of the fact that we explicitly support using other things as the `.nx-card__content` children besides `.nx-card__callout` and `.nx-card__text`. It's only really a 2-column layout when the `.nx-card__content` happens to have two children.

I'm not sure where the concern lays. Are you saying that it's named poorly, or that you'd like to see an example that uses something other than call-out and text?

Technically it would support a number of columns equal to the number of children of `content`, should we describe that instead?

Reply:
> > I'm mildly concerned about the naming of this class in light of the fact that we explicitly support using other things as the `.nx-card__content` children besides `.nx-card__callout` and `.nx-card__text`. It's only really a 2-column layout when the `.nx-card__content` happens to have two children.
> 
> I'm not sure where the concern lays. Are you saying that it's named poorly, or that you'd like to see an example that uses something other than call-out and text?
> 
> Technically it would support a number of columns equal to the number of children of `content`, should we describe that instead?

I changed it to the more generic `--columns`: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/182df9d6645a4b1d575b7f9113a29399bc8c5e83